app/assets/javascripts/order.js

```diff
@@ -4,8 +4,12 @@ $(document).ready(function() {
   $("#new_order").submit(function(event) {
     var $form = $(this);
     $form.find("button").prop("disabled", true);
-    Stripe.card.createToken($form, stripeResponseHandler);
-    return false;
+    if $("card_number").length;
```

Comment:
Expected '(' and instead saw '$'.<br>Mixed double and single quotes.<br>Expected ')' to match '{b}' from line 7 and instead saw ';'.